dds/idl/be_global.cpp

```diff
@@ -319,6 +325,9 @@ BE_GlobalData::parse_args(long& i, char** av)
   static const char EXPORT_FLAG[] = "--export=";
   static const size_t EXPORT_FLAG_SIZE = sizeof(EXPORT_FLAG) - 1;
 
+  static const char DEFAULT_TS_MACRO[] = "--default-nested";
+  static const size_t SZ_DEFAULT_TS_MACRO = sizeof(DEFAULT_TS_MACRO) - 1;
```

Comment:
Any chance we can be consistent (either direction) for how we name our parameters and their sizes? (see above)

Reply:
I have some changes I've not pushed yet where I fixed this.